apps/fluent-tester/src/TestComponents/Chip/ChipDefault.tsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import React, { useCallback } from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { Chip } from '@fluentui-react-native/chip';
+import type { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
+import { Switch } from '@fluentui-react-native/switch';
+import { Text } from '@fluentui-react-native/text';
+
+import { svgProps } from '../Common/iconExamples';
+
+export const ChipDefault: React.FunctionComponent = () => {
+  const [showCloseIcon, setShowCloseIcon] = React.useState<boolean>(false);
+  const toggleActivityState = useCallback((_e: InteractionEvent, isActive: boolean) => setShowCloseIcon(isActive), []);
+
+  return (
+    <View>
+      <Switch label={'Show Close Icon'} defaultChecked={showCloseIcon} onChange={toggleActivityState} />
+      <View style={styles.rowStyle}>
+        <Chip size="small" style={styles.rowGap} showCloseIcon={showCloseIcon} closeIconOnPress={() => console.log('Close icon clicked!')}>
```

Comment:
can we demonstrate something on close click, console log doesn't make sense for tester app ?
 what does close icon click do ? if it's actionable on the consumer to provide the implementation we can add feedback. 
 
 As chip is android only we can display this feedback : https://reactnative.dev/docs/toastandroid as toast.

Reply:
Same for other places.

Reply:
updated